execution_engine/src/core/execution/executor.rs

```diff
@@ -485,86 +485,6 @@ impl Executor {
         ret
     }
 
-    /// Used to execute arbitrary wasm; necessary for running system contract installers / upgraders
-    /// This is not meant to be used for executing system contracts.
-    pub fn exec_wasm_direct<R, T>(
```

Comment:
Was this method unused all along or is it possible to remove it only as part of this PR?